Assets/Scripts/UIScripts/OptionsMenu.cs

```diff
@@ -1,27 +1,26 @@
+using System;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Audio;
+using UnityEngine.SceneManagement;
 
-public class SettingMenu : MonoBehaviour
+public class OptionsMenu : MonoBehaviour
 {
     public AudioMixer audioMixer;
-
     public TMPro.TMP_Dropdown resolutionDropdown;
+    public Action onBackToMainMenu;
 
-    Resolution[] _resolutions;
+    private Resolution[] _resolutions;
 
     private void Start()
     {
         _resolutions = Screen.resolutions;
-
         resolutionDropdown.ClearOptions();
-
         var options = new List<string>();
```

Comment:
```suggestion
        var options = _resolutions.Select(x => x.width + " x " + x.height).ToList();
        var c = Screen.currentResolution;
        var i = Array.FindIndex(_resolutions, x => x.width == c.width && x.height == c.height);
        var currentResolutionIndex = i < 0 ? 0 : i;
```

То же самое, через Linq

Reply:
Сделано